Digipost/POSLetterViewController.m

```diff
@@ -167,31 +166,41 @@ - (void)viewDidLoad
 - (void)shouldValidateOpeningReceipt:(POSAttachment *)attachment
 {
     POSDocument *document = attachment.document;
-    [[APIClient sharedClient] validateOpeningReceipt:attachment success:^{
```

Comment:
All changes after this seems to be triggered by an update in Clang-format http://clang.llvm.org/docs/ClangFormat.html
It's used to keep the objc code nicely formatted (though some times changes alot of the code if a new rule is added)
Optimally code changes like that should be committed as own commits to not hide their actual purpose